### ISSUE TITLE
fix(html-spec): add missing HTML Living Standard attributes (closedby, writingsuggestions, etc.)

### ISSUE THREAD

**Issue by YusukeHirao** [markuplint/markuplint#3590]
## Describe the bug

Several recently added HTML Living Standard attributes are missing from the markuplint spec data, causing `invalid-attr` false positives on valid HTML.

### Missing attributes

| Element | Attribute | Spec reference |
|---------|-----------|---------------|
| `dialog` | `closedby` | https://html.spec.whatwg.org/multipage/interactive-elements.html#attr-dialog-closedby |
| `*` (global) | `writingsuggestions` | https://html.spec.whatwg.org/multipage/interaction.html#attr-writingsuggestions |
| `details` | `name` | https://html.spec.whatwg.org/multipage/interactive-elements.html#attr-details-name |
| `link[rel=modulepreload]` | `as` | Valid values include `json`, `worker` in addition to existing values |

## The current behavior

```
The "closedby" attribute is not allowed
The "writingsuggestions" attribute is not allowed
```

## The expected behavior

No violations reported.

## Impact

Causes ~10 false positives in the nu-html-checker compatibility benchmark.

## Affected package

`@markuplint/html-spec` — element/attribute spec data

**Comment by YusukeHirao** [comment]
Closing — all attributes listed in this issue already exist in the spec data (added via MDN scraping):

- `dialog[closedby]` ✅ present
- `writingsuggestions` (global) ✅ present in `#HTMLGlobalAttrs`
- `details[name]` ✅ present
- `link[as]` ✅ present (includes modulepreload values)

The false positives in the nu-html-checker benchmark were caused by the `lang=""` issue (#3585), not missing attributes. With #3585 merged (PR #3642), these are resolved.
